packages/platforms/delta/x86-64/x86-64-delta-ag7648/platform-config/r0/src/python/x86_64_delta_ag7648_r0/__init__.py

```diff
@@ -0,0 +1,31 @@
+from onl.platform.base import *
+from onl.platform.delta import *
+
+class OnlPlatform_x86_64_delta_ag7648_r0(OnlPlatformDelta):
```

Comment:
Please inherit from OnlPlatformPortConfig_48x10_6x40 instead of setting the member variables yourself. 
See https://github.com/opencomputeproject/OpenNetworkLinux/blob/master/packages/platforms/accton/x86-64/x86-64-accton-as5712-54x/platform-config/r0/src/python/x86_64_accton_as5712_54x_r0/__init__.py#L4 as an example.